mosec/mixin/plasma_worker.py

```diff
@@ -74,13 +74,13 @@ def _get_client(self):
     def serialize_ipc(self, data: Any) -> bytes:
         """Save the data to the plasma server and return the id."""
         client = self._get_client()
-        object_id = client.put(data)
+        object_id = client.put(super().serialize_ipc(data))
```

Comment:
Yes, you can also replace it with `class Inf(PlasmaShmIPCMixin, NumBinIPCMixin, Worker)` to use the numbin serde. Although this is reliable, it might be a little complex to nest more Mixins.

Reply:
I feel that the choice of IPC should replace one another, instead of being chained?

Reply:
Yep, I think it will be slightly slower since serde the bytes will also take some time. But in this case, we can make it correct (at least for the common objects).

Reply:
> Yep, I think it will be slightly slower since serde the bytes will also take some time. But in this case, we can make it correct (at least for the common objects).

Any example where serializing common objects might fail without this nested structure?

Reply:
> I feel that the choice of IPC should replace one another, instead of being chained?

Refer to https://github.com/mosecorg/mosec/pull/324

It's hard to decide. I think this is the main reason that we create the IPC Wrapper abstraction. But actually, it's rarely used except for this one particular reason.